### PR TITLE
Some fix and improvement for Luke <3

### PR DESCRIPTION
**Pull request by CeebDev**
As i told you on my stream, this is a lil PR to update map and fix lil oopsies
There is still some problems on the mdt and will try to make another PR if i have time

### CODE REVIEW

File: server/officers.lua [hunk 1]
```diff
@@ -16,8 +16,8 @@ SetInterval(function()
 
     for _, officer in pairs(activeOfficers) do
         local coords = GetEntityCoords(officer.ped)
-        officer.position[1] = coords.y
-        officer.position[2] = coords.x
+        officer.position[1] = coords.x
+        officer.position[2] = coords.y
```
Comment (LukeWasTakenn) [inline server/officers.lua:20]: Coords were reversed in [this](https://github.com/overextended/ox_mdt/pull/27) PR. Does this reverse them again?

Comment (CeebDev) [reply server/officers.lua:20]: Not the same file, it was a mistake everyhere, and this is what we actually use on prod with the map update

**Review by LukeWasTakenn — APPROVED** [review]

**Comment by CeebDev** [comment]
we can also get rid of old map image, not on my PC anymore

**Comment by LukeWasTakenn** [comment]
LGTM, thank you!

**Comment by LukeWasTakenn** [comment]
Yeah I also remember the leaflet coords being reversed, @CeebDev does this not happen for you? 🤔

**Comment by CeebDev** [comment]
I will check but it was working on my side when i have made this change.

**Comment by CeebDev** [comment]
But, map is good ? It's just the blip on game map ? Not in the MDT map ? I dont understand the message from @MauroNr1

**Comment by MauroNr1** [comment]
Yea its just the blip on the game map. The MDT map is correct but the game map isn't.

**Comment by CeebDev** [comment]
> Yea its just the blip on the game map. The MDT map is correct but the game map isn't.

Then make a PR to invert only on the blip creation

**Comment by MauroNr1** [comment]
Done and added some other minor fixes regarding blips https://github.com/overextended/ox_mdt/pull/30

**Pull request merged**